data/human/humanity interludes.txt

```diff
@@ -0,0 +1,192 @@
+			`	Below the message is a pair of buttons offering the option to accept or decline the invitation.`
+			choice
+				`	Accept.`
+					accept
+				`	Decline.`
+					decline
+	on decline
+		event "humanity: Broadsider Outfit" 10 30
+	on abort
+		event "humanity: Broadsider Outfit" 10 30
+
+
+
+event "humanity: Broadsider Outfit"
+	outfitter "Lovelace Advanced"
+		add "Broadsider"
+		add "Broadsider Storage"
+		add "Broadsider Slug"
+	outfitter "Ammo North"
+		add "Broadsider Storage"
+		add "Broadsider Slug"
+
+
+
+mission "Humanity: Broadsider 2"
+	name "Naval Research Consultation."
+	description "Observe the weapons test, land on Pilot, and then return to Geminus."
+	landing
+	to offer
+		has "Humanity: Broadsider 1: done"
+	source "Geminus"
+	stopover "Pilot" 
+	destination
+		planet "Geminus"
+	on offer
+		conversation
+			`On Geminus you are met at the spaceport by Andre, who turns out to be almost a caricature of the classic scientist: Thin, balding, a slightly frazzled look as if he was recently charged with static electricity, and a white lab coat. Unlike the caricature scientist, however, he radiates an air of calm confidence at odds with his appearance, and the military fatigues and combat boots suggest that he is Navy first, researcher second.`
+			`	"Welcome to Geminus, Captain <last>!" It is a pleasure to meet you." He nods politely, and gestures for you to follow him. A few hallways and checkpoints later, and you find yourself in a conference room. "Thank you for accepting our request to consult on this project. Now, I'm sure you have some questions, so I'll get straight to it." He taps a button, and a projector lights up, displaying an image of a Sestor fleet. "Several months ago you were instrumental in defeating a flood of robotic alien warships that swept across Northern Republic space." He flicks through several slides showing long range photography, scans, and video footage of various Sestor ships. "While we have known about the Hai for quite some time, it was quite a shock to be invaded by a hitherto unknown race. After so many years of peaceful coexistence with the Hai and the Quarg, a warlike group caught us by surprise."`
+			`	He flips through a few more slides, these ones show the broken husks of Sestor ships, wrecks, and some salvaged outfits spread out on a tarmac. "As a result, Parliament has seen fit to significantly increase the Navy's R&D budget. This, combined with the salvage and debris we have collected from these 'Sestor' ships has allowed several of our long-standing projects to accelerate." He turns towards you. "This is where you come in: You have more combat experience against the Sestor than any other captain we know of. We would like you to inspect a new weapon we have developed, observe a weapons demonstration test, and then give us your thoughts on how effective it will be against the Sestor. So, do you accept?"`
```

Comment:
```suggestion
			`	He flips through a few more slides. These ones show the broken husks of Sestor ships, wrecks, and some salvaged outfits spread out on a tarmac. "As a result, Parliament has seen fit to significantly increase the Navy's R&D budget. This, combined with the salvage and debris we have collected from these 'Sestor' ships has allowed several of our long-standing projects to accelerate." He turns towards you. "This is where you come in: you have more combat experience against the Sestor than any other captain we know of. We would like you to inspect a new weapon we have developed, observe a weapons demonstration test, and then give us your thoughts on how effective it will be against the Sestor. So, do you accept?"`
```